### ISSUE TITLE
Can't find file in evaluation

### ISSUE THREAD

**Issue by luzerochen** [hyperconnect/TC-ResNet#13]
Hi, I run the code and meet an error.  

![image](https://user-images.githubusercontent.com/54262164/82857220-f842b280-9f42-11ea-811d-caebf12d12a0.png)
I didn't find when the file was created. Does this file need to be created manually?

Sincerely,
Chen

**Comment by GEORGJOSE007** [comment]
Hi, first you need to train the models as per the instructions and then this folder will be created while training. Model with the best validation accuracy will be saved to this folder during training.

**Comment by luzerochen** [comment]
> Hi, first you need to train the models as per the instructions and then this folder will be created while training. Model with the best validation accuracy will be saved to this folder during training.

Yes. I just run the '.sh' file in ‘TC-ResNet/scripts/commands/’ 
./scripts/commands/DSCNNSModel-0_mfcc_10_4020_0.0000_adam_l3.sh
It seems the code doesn't create the file. Do you know where the code will create file. I can not find it.

Thanks for your reply

**Comment by GEORGJOSE007** [comment]
Hi,
 If the training and validation has completed without errors, then it should be created in the path mentioned above. In some cases, it was noticed that sleep time of 5 secs is not sufficient enough causing validation to not happen. Increasing sleep time to 50 secs in shell script helped
